blog/_posts/2017-10-05-TeamGemini-Happy-ending-yet-not-the-end.md

```diff
@@ -0,0 +1,55 @@
+
+At one such opensource meetup, [Akarsha](https://twitter.com/akarsha98) also met the co-founder of her early-stage startup ‘[Invadex](invadex.in)’ which she is going to represent in the Alpha program of [Web-summit](www.websummit.com) this November which is regarded as **“The Largest Tech Conference in the World”**. And because of our exposure with Scikit-Image, [Ramya](https://twitter.com/notrandomatalll) decided to pursue research with the Image Analysis and Biometrics lab of our college. This is how Rgsoc helped us kickstart our careers and grow a lot as individuals. 
+
+<iframe src="https://giphy.com/embed/l3q2u6MXJJEKiTZIY" width="700" height="480" style="float:initial" frameBorder="0" class="giphy-embed" allowFullScreen></iframe>
+
+A big big thanks to Rgsoc for giving us the opportunity and for their constant support.
+
+
+## Our journey
+
+Well, in short, the most amazing summer till date ! We spent nights working on a problem, had arguments, submitted proposals, shifted projects, volunteered at conference, met siraj raval, learned new UI libraries, gave talks, stalked data-science people together and what not. These were a small fraction of the awesome memories this summer gave us.
+
+Initially starting with Susi AI server as our project, we were happy to work on artificial intelligence for chatbots, though rule based. With no timeline in hand, we worked on random issues and created skills for susi. One month gone, we lost the excitement without the inputs from the mentors and looking at the situation, orga team changed the project for us. It was a blessing in disguise since we got to work with Scikit-image. It is a python library for Image analysis. We were so lucky to get such amazing mentors. They assigned us the task of analysing the performance of fast_histogram wrt numpy histogram and benchmarking them for all instances in the entire repo which we successfully completed. 
+
+Things we learnt in these 3 months:
+
+- Proficiency with git
+- Jupyter notebook
+- Virtualisation
+- Image analysis in Python
+- Data extraction using API’s
+- Working of AI chatbots
+- React JS
+- Using Docker and Heroku
+
+## Our journey ahead
+
+This is not the end, this is definitely not the end!! We are planning to continue to contribute to Scikit-Image. Apart from that, we are super excited to attend [Websummit](https://websummit.com) this November. 
+Andd we are *Organizing [Django Girls Lisbon](https://djangogirls.org/lisbon) on 5th November* to let more girls fall in love with programming and open-source.. 
```

Comment:
Typo for 'And' is still there. Or do you mean any internet abbreviation here which I don't understand?